src/main/resources/application-RECEIVER.properties

```diff
@@ -99,16 +99,16 @@ information.model.version=@information.model.version@
 application.basedir=@application.basedir@
 
 # Enable WebSocket over Https -> Disable Idscp to use!
-application.websocket.isEnabled=false
+application.websocket.isEnabled=true
```

Comment:
Should we put those "as default" to be false, to use http flow, not wss?

Reply:
Sure, no problem.